advocacy_docs/pg_extensions/system_stats/index.mdx

```diff
@@ -0,0 +1,17 @@
+---
+title: 'system_stats'
```

Comment:
Double-check this - repo suggests it may be properly named System Statistics or EDB System Statistics Extension.

Reply:
That does seem to be it's extension name and not its formal name. It just turned up in the extensions table.